blocks/small-promo-block/features/small-promo/default.jsx

```diff
@@ -86,6 +86,21 @@ const SmallPromo = ({ customFields }) => {
 	}`,
 		}) || null;
 
+	const resizedImage =
```

Comment:
@blakeganderson , another issue I noticed, is that the filter on this block is incorrect and making it so lead art galleries and videos don't show any images.

Above, the lead art filter is this:
```
lead_art {
  _id
  type
  url
  auth {
    ${RESIZER_TOKEN_VERSION}
  }
}
```

But I should match the other promos, like this:
```
lead_art {
  embed_html
  type
  promo_items {
    basic {
      _id
      type
      url
      auth {
        ${RESIZER_TOKEN_VERSION}
      }
    }
  }
}
```